src/lit-html.ts

```diff
@@ -268,23 +268,34 @@ export class NodePart extends Part {
   setValue(value: any): void {
     value = this._getValue(value);
 
-    if (value instanceof Node) {
-      this._previousValue = this._setNode(value);
+    if (value === undefined ||
```

Comment:
For reference, [this](https://github.com/Polymer/polymer/blob/master/lib/mixins/property-accessors.html#L607) is the dirty check as implemented by Polymer